data-es2016plus.js

```diff
@@ -3453,10 +3453,10 @@ exports.tests = [
       */},
         res: {
           ie11: false,
-          chrome117: false,
-          node20_0: false,
+          chrome118: false,
+          node21_0: false,
```

Comment:
```suggestion
          node21_0: false,
          node22_0: true,
```
I can confirm Node.js 22 installed from `nvm` supports this feature.